src/core/sys/posix/sys/mman.d

```diff
@@ -455,35 +420,8 @@ int mlock(in void*, size_t);
 int munlock(in void*, size_t);
 */
 
-version( linux )
-{
-    int mlock(in void*, size_t);
-    int munlock(in void*, size_t);
-}
-else version( OSX )
-{
-    int mlock(in void*, size_t);
-    int munlock(in void*, size_t);
-}
-else version( FreeBSD )
-{
-    int mlock(in void*, size_t);
-    int munlock(in void*, size_t);
-}
-else version (Solaris)
-{
-    int mlock(in void*, size_t);
-    int munlock(in void*, size_t);
-}
-else version (Android)
-{
-    int mlock(in void*, size_t);
-    int munlock(in void*, size_t);
-}
-else
-{
-    static assert(false, "Unsupported platform");
-}
+int mlock(in void*, size_t);
+int munlock(in void*, size_t);
```

Comment:
[Defined this way in the POSIX standard](http://pubs.opengroup.org/onlinepubs/9699919799/functions/mlock.html).